sql/src/main/java/org/apache/druid/sql/calcite/expression/builtin/QueryLookupOperatorConversion.java

```diff
@@ -91,4 +103,18 @@ public DruidExpression toDruidExpression(
         }
     );
   }
+
+  private String getReplaceMissingValueWith(
+      final List<DruidExpression> inputExpressions,
+      final PlannerContext plannerContext
+  )
+  {
+    if (inputExpressions.size() > 2) {
+      final Expr missingValExpr = plannerContext.parseExpression(inputExpressions.get(2).getExpression());
+      if (missingValExpr.isLiteral()) {
+        return missingValExpr.getLiteralValue().toString();
+      }
```

Comment:
this should maybe have a similar validation to the native layer to make sure that the argument doesn't silently become null in the event that the expression is null

Reply:
Added a test in CalciteQueryTest to make sure that null is wired up correctly to native extraction fn.  
modified the returned value to`(String) missingValExpr.getLiteralValue()`